android/app/src/main/java/safetyfirst/androidapp/safetyfirstcontroller/fragments/ViewContactFragment.java

```diff
@@ -1,43 +1,42 @@
-package safetyfirst.androidapp.safetyfirstcontroller;
+package safetyfirst.androidapp.safetyfirstcontroller.fragments;
 
-import androidx.appcompat.app.AppCompatActivity;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
 
 import android.os.Bundle;
+import android.view.LayoutInflater;
 import android.view.View;
+import android.view.ViewGroup;
 import android.widget.AdapterView;
 import android.widget.ArrayAdapter;
 import android.widget.Button;
 import android.widget.ListView;
 import android.widget.Toast;
 
-import java.util.Objects;
-
 import safetyfirst.androidapp.safetyfirstcontroller.Data.DataBaseHelper;
 import safetyfirst.androidapp.safetyfirstcontroller.Model.EmergencyContact;
+import safetyfirst.androidapp.safetyfirstcontroller.R;
 
-public class ContactList extends AppCompatActivity {
-
-
+public class ViewContactFragment extends Fragment {
     //Database
     ArrayAdapter customerArrayAdapter;
     DataBaseHelper dataBaseHelper;
+    //ListView lv_contactList;
```

Comment:
Why is this commented out?

Reply:
Has been removed, the variable in use is written below this.